assets/src/css/frontend/form-grid.scss

```diff
@@ -0,0 +1,39 @@
+.give-form-grid-card__body {
+    align-items: center;
```

Comment:
The position of the donate button is kind of inconsistent depending the contents of the form:

![image](https://user-images.githubusercontent.com/7819986/166330244-77f23b3f-adba-4ca7-93b1-06de16cb97fa.png)

Maybe adding `justify-content: space-evenly` or something like that would help?

Reply:
Thanks for the suggestion, implemented 48538b10 
That helped :raised_hands: